src/rz/angular/effects/standalone-pipe/snapshots/index-output.ts.snap

```diff
@@ -0,0 +1 @@
+export * from "./standalone-pipe";
```

Comment:
we are moving away from using snapshots. we will remove this and use a string only